src/devices/device_models/common/range_constraint.jl

```diff
@@ -0,0 +1,86 @@
+function device_range(ps_m::CanonicalModel, 
+                        range_data::Array{Tuple{String,NamedTuple{(:min, :max),Tuple{Float64,Float64}}},1}, 
```

Comment:
Not use `Array{Tuple{String,NamedTuple{(:min, :max),Tuple{Float64,Float64}}},1}` since it is hard to read.

1) Use a alias for the tuple component
2) Prefer Vector{T} instead of Array{T, 1}

```
const MinMaxRange = Tuple{String,NamedTuple{(:min, :max),Tuple{Float64,Float64}}}
...
range_data::Vector{MinMaxRange}
...
```

Reply:
Fixed via a351de5